modules/convert/issue.go

```diff
@@ -25,6 +28,9 @@ func ToAPIIssue(issue *models.Issue) *api.Issue {
 	if err := issue.LoadRepo(); err != nil {
 		return &api.Issue{}
 	}
+	if err := issue.Repo.GetOwner(); err != nil {
```

Comment:
I think `err := issue.Repo.GetOwner();` is misleading, as this implies that `err` would be the owner.
After what I have seen, this function should most likely be renamed to `LoadOwner`, because that's what it does.
Returning only an error makes much more sense then.
When renaming it, its function comment should be changed as well as that is simply incorrect by now.
Could it be that that is a leftover from when the owner was initialized eagerly?